kymatio/scattering2d/frontend/torch_frontend.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, J, shape, L=8, max_order=2, pre_pad=False, backend='torch'):
         self.register_filters()
 
     def register_single_filter(self, v, n):
-        current_filter = torch.from_numpy(v).unsqueeze(-1)
+        current_filter = torch.from_numpy(v).unsqueeze(-1).type_as(torch.zeros(1))
```

Comment:
funny trick again

Reply:
Yeah don't like this either.

Reply:
Well, it's not that useless actually: imagine if an user needs to work in float64/float32

Reply:
So now they have to be able to change the default `dtype` in Torch as well? Sounds very difficult.

Reply:
fair enough